test/logger-usage/src/main/java/org/elasticsearch/test/loggerusage/ESLoggerUsageChecker.java

```diff
@@ -82,7 +82,7 @@ private static void checkLoggerUsage(Consumer<WrongLoggerUsage> wrongUsageCallba
             Files.walkFileTree(root, new SimpleFileVisitor<Path>() {
                 @Override
                 public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
-                    if (Files.isRegularFile(file) && file.endsWith(".class")) {
```

Comment:
This is the reason that the logger usage checker was disabled. `file.endsWith(".class")` checks if the last element of the path is exactly `.class`, i.e. for a path `foo/Bar.class` it would check that `Bar.class` is equal to `.class` :-(